packs/azure/pack.yaml

```diff
@@ -9,6 +10,6 @@ keywords:
   - servers
   - virtual machines
   - azure virtual machines
-version : 0.1
-author : st2-dev
-email : info@stackstorm.com
+version : 0.1.0
+author : StackStorm, Inc.
+email : info@stackstorm.com
```

Comment:
@emedvedev I agree, I will push a change which adds line break back.

I actually only wanted to truncate additional empty lines at the end of the file (some files had them), but accidentally removed all the line breaks.